### PR TITLE
Add support for numeric source fields

### PR DESCRIPTION
**Pull request by nurges**
Add support for creating slugs for various id fields that are integer type.

### CODE REVIEW

File: lib/ecto_autoslug_field/slug_generator.ex [hunk 1]
```diff
@@ -88,7 +88,8 @@ defmodule EctoAutoslugField.SlugGenerator do
   defp get_field_data(_, source, _) when is_binary(source), do: source
 
   defp has_value?(nil), do: false
-  defp has_value?(string) do
-    String.strip(string) != ""
+  defp has_value?(string) when is_binary(string) do
+    String.trim(string) != ""
   end
+  defp has_value?(_), do: true
```
Comment (sobolevn) [inline lib/ecto_autoslug_field/slug_generator.ex]: Could you please test this function with some corner cases?
I am not quite sure about this line, how it will behave with different types of data?

Comment (nurges) [reply lib/ecto_autoslug_field/slug_generator.ex]: I'll try to add some more test cases.
I just needed to slug a simple integer field, but I can see DateTime as a valid option too.

One problem is how to define if a number has a value. Is 0 a value? I suppose so.
So I'd return 'true' from this function for anything except strings.

Comment (nurges) [reply lib/ecto_autoslug_field/slug_generator.ex]: I added several test cases with multiple field types and they all seem to work ok.

**Comment by sobolevn** [comment]
Thanks! Nice work.
I will release this shortly.
